server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -398,6 +399,23 @@ public int queryTimeoutSeconds() {
         return queryTimeoutSeconds;
     }
 
+    public ConnectionInfo simplify() {
```

Comment:
prefer `toConnectionInfo`, otherwise the user can not get the meaning of this method

Reply:
renamed this.